ext/mysql2/client.c

```diff
@@ -1155,12 +1157,12 @@ static VALUE rb_mysql_client_session_track(VALUE self, VALUE type) {
  * if it was an UPDATE, DELETE, or INSERT.
  */
 static VALUE rb_mysql_client_affected_rows(VALUE self) {
-  my_ulonglong retVal;
+  uint64_t retVal;
```

Comment:
Ok, I see the type changed between MySQL 5.7 and 8.0:
https://dev.mysql.com/doc/c-api/8.0/en/mysql-affected-rows.html
```
uint64_t
mysql_affected_rows(MYSQL *mysql)
```
https://dev.mysql.com/doc/c-api/5.7/en/mysql-affected-rows.html
```
my_ulonglong
mysql_affected_rows(MYSQL *mysql)
```